tests/src/python/test_qgsgeometry.py

```diff
@@ -4244,42 +4244,42 @@ def testGeometryDisplayString(self):
 
     def testDeleteVertexCircularString(self):
 
-        wkt = "CircularString ((0 0,1 1,2 0))"
```

Comment:
Can you also add this one to TestQgsGeometry::wktParser()

Reply:
Done